OAuth2-UserIdentity/Web.config

```diff
@@ -16,8 +16,8 @@
     <add key="webpages:Enabled" value="false" />
     <add key="ClientValidationEnabled" value="true" />
     <add key="UnobtrusiveJavaScriptEnabled" value="true" />
-    <add key="ida:ClientId" value="f8e269ae-8eb2-4c73-8d04-7640f510fda1" />
-    <add key="ida:AppKey" value="sypghJlPLg3lZHW6HRXoFEUo9ADFwbVBwyRYDa9PxDc=" />
+    <add key="ida:ClientId" value="[Enter client ID as obtained from Azure Portal, e.g. 82692da5-a86f-44c9-9d53-2f88d52b478b]" />
+    <add key="ida:AppKey" value="[Enter app key as obtained from Azure Portal, e.g. rZJJ9bHSi/cYnYwmQFxLYDn/6EfnrnIfKoNzv9NKgbo=]" />
     <add key="ida:Tenant" value="kalyankrishna.in" />
```

Comment:
Do you keep the tenant to your own tenant ?

Reply:
This was an oversight, fixed